docs/README.md

```diff
@@ -0,0 +1,77 @@
+# Codeshare
+
+CodeShare is a web-based real-time code sharing tool. It empowers developers to collaborate remotely during hackathons, contests or for a project. It can also be potentially used in mentorship programs to live share code. The tool also compiles and runs code.
+
+## Team members:
+1. Rohit M P
+2. Shanthanu S Rai
+3. Ameya N Deshpande
+4. Harshvardhan R
+5. Vaibhav Puri
+
+## Objectives:
+
+Following were the objectives:
+1. Compare Codemiror vs Monaco and integrate either of them.
+2. To create a server-based system using Websockets (Channel).
+3. Display the users and their respective cursors on the editor.
+4. Implement CRDT functions to handle real time problems 
```

Comment:
- Add link for CRDT
- maybe reframe to "to handle consistency among users"

Reply:
Done